tests/utils_tests/test_timezone.py

```diff
@@ -1,14 +1,38 @@
 import datetime
+import unittest
 from unittest import mock
 
 import pytz
 
+try:
+    import zoneinfo
+except ImportError:
+    try:
+        from backports import zoneinfo
+    except ImportError:
+        zoneinfo = None
+
 from django.test import SimpleTestCase, override_settings
 from django.utils import timezone
 
-CET = pytz.timezone("Europe/Paris")
+PARIS = pytz.timezone('Europe/Paris')
 EAT = timezone.get_fixed_timezone(180)      # Africa/Nairobi
 ICT = timezone.get_fixed_timezone(420)      # Asia/Bangkok
+UTC = datetime.timezone.utc
+
+HAS_ZONEINFO = zoneinfo is not None
+PARIS_IMPLS = (PARIS,)
+
+if not HAS_ZONEINFO:
+    PARIS_ZI = None
+    PARIS_IMPLS = (PARIS,)
+
+    def needs_zoneinfo(f):
+        return f
```

Comment:
isn't this back to front? The skip definition should be here, for if the user doesn't have zoneinfo?

Reply:
Oops, good catch!